client/ayon_core/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -88,8 +98,25 @@ def create_instance_in_context_from_existing(self, data):
         return instance
 
     def get_instance_attr_defs(self):
-
+        layer_stack_channel_enum = ["BaseColor", "Metallic", "Roughness",
+                                    "Normal", "Height", "Specular",
+                                    "SpecularEdgeColor", "Emissive", "Opacity",
+                                    "Displacement", "Glossiness", "Anisotropylevel",
+                                    "AO", "Anisotropyangle", "Transmissive",
+                                    "Reflection", "Diffuse", "Ior",
+                                    "Specularlevel", "BlendingMask", "Translucency",
+                                    "Scattering", "ScatterColor", "SheenOpacity",
+                                    "SheenRoughness", "SheenColor", "CoatOpacity",
+                                    "CoatColor", "CoatRoughness", "CoatSpecularLevel",
+                                    "CoatNormal"]
```

Comment:
Should we also present this to the artist with nice names?

Reply:
Do you have any suggestions for this? I follow the documentation and come up with this one. Also they look the same in the output templates.
![image](https://github.com/ynput/ayon-core/assets/64118225/e2826ed2-6fbf-455b-bf49-3c40bdd1c37f)

Reply:
I think that ref is perfect - that'd just mean adding spaces to e.g. `Base color` etc.?

Reply:
> I think that ref is perfect - that'd just mean adding spaces to e.g. `Base color` etc.?

we need to use items instead of this.

Reply:
Yes, we'd likely need to specify a `dict` from nice name to backend value. Correct.